sc/src/main/webapp/WEB-INF/tiles/views.xml

```diff
@@ -40,4 +40,9 @@
         <put-attribute name="title" value=""/>
         <put-attribute name="content" value="/WEB-INF/jsp/tiles/views/static-page.jsp"/>
     </definition>
+
+    <definition name="hca-landing-page" extends="base">
+        <put-attribute name="title" value="Experiment &lt; "/>
```

Comment:
Replace Experiment with an appropriate title in this page. As the view is extended it will be prepended to form the `<title>` tag. Have a look at the rest of the views to see other examples.